packages/react-charting/etc/react-charting.api.md

```diff
@@ -787,13 +787,14 @@ export interface ILegendsProps {
     canSelectMultipleLegends?: boolean;
     centerLegends?: boolean;
     className?: string;
+    defaultSelectedLegends?: string[];
```

Comment:
defaultSelectedLegend prop is missing here

Reply:
update